.circleci/config.yml

```
@@ -439,6 +440,40 @@ jobs:
             find /tmp/${DATASET}/derivatives \( -path */figures -o -path */log \) -prune -o -name "*" -print | sed s+/tmp/${DATASET}/derivatives/++ | sort > /tmp/${DATASET}/test/outputs.out
             diff /tmp/src/fmriprep/.circleci/${CHECK_OUTPUTS_FILE} /tmp/${DATASET}/test/outputs.out
             exit $?
+      - run:
+          name: Prepare YODA dataset
+          command: |
+            mkdir -p /tmp/${DATASET}/yoda/sourcedata
+            cp -r /tmp/${DATASET}/freesurfer /tmp/${DATASET}/yoda/sourcedata/freesurfer
+      - run:
+          name: Re-run with YODA mode
+          no_output_timeout: 2h
+          command: |
+            FASTRACK_ARG="--anat-derivatives /tmp/${DATASET}/smriprep-0.6"
+            if [ -f /tmp/.nofasttrack ]; then
+                FASTRACK_ARG=""
+            fi
+            fmriprep-docker -i nipreps/fmriprep:latest \
+                -e FMRIPREP_DEV 1 -u $(id -u) \
+                --network none \
+                --config $PWD/nipype.cfg -w /tmp/${DATASET}/work_yoda \
+                /tmp/data/${DATASET}/ /tmp/${DATASET}/yoda participant \
+                ${FASTRACK_ARG} \
+                --output-layout bids \
+                --sloppy --write-graph --use-syn-sdc --mem_mb 4096 \
+                --output-spaces MNI152NLin2009cAsym \
+                --nthreads 2 -vv
+      - run:
+          name: Checking outputs of partial fMRIPrep run
+          command: |
+            mkdir -p /tmp/${DATASET}/test
+            CHECK_OUTPUTS_FILE="${DATASET}_yoda_fasttrack_outputs.txt"
+            if [ -f /tmp/.nofasttrack ]; then
+                CHECK_OUTPUTS_FILE="${DATASET}_yoda_outputs.txt"
+            fi
+            find /tmp/${DATASET}/yoda \( -path */figures -o -path */log -path */sourcedata \) -prune -o -name "*" -print | sed s+/tmp/${DATASET}/++ | sort > /tmp/${DATASET}/test/outputs.out
```

@oesteban Since you wrote the original `find` command, perhaps you know why this isn't ignoring `sourcedata`?

---

I think you were missing an `-o`:

```suggestion
            find /tmp/${DATASET}/yoda \( -path */figures -o -path */log -o -path */sourcedata \) -prune -o -name "*" -print | sed s+/tmp/${DATASET}/++ | sort > /tmp/${DATASET}/test/outputs.out
```